ui/app/AppLayouts/Chat/popups/ProfilePopup.qml

```diff
@@ -89,12 +89,23 @@ StatusModal {
             anchors.top: parent.top
             width: parent.width
 
+            // Blocked User Status Bar
+            StatusBanner {
+                id: blockedUsrBar               
+                width: parent.width
+                visible: isBlocked
+                backgroundColor: Style.current.tenPercentRed
+                bordersColor: Style.current.twentyPercentRed
+                fontColor: Style.current.red
```

Comment:
Instead of assigning concrete colors here, `StatusBanner` should rather have variants for `Success` and `Danger` (and possibly others if needed). Mentioned it here: https://github.com/status-im/StatusQ/commit/c56dcd9eb47ad8e22d24257b53bbc12ba407357f#commitcomment-63733961